config/routes.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+Decidim::Blogs::Engine.routes.draw do
```

Comment:
These routes should be in the dedicated Engine to be homogeneous with community modules

Reply:
Here is an example of added routes in an existing Decidim module 
https://github.com/mainio/decidim-module-budgeting_pipeline/blob/7098de1228fbd073a42ccaec9ba06a421b22ce3e/lib/decidim/budgeting_pipeline/engine.rb#L53